sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/models/_models_py3.py

```diff
@@ -216,87 +216,87 @@ def __init__(
-    :type confidence_score: float
+    :keyword category: Required. The entity category.
+    :paramtype category: str
+    :keyword text: Required. The predicted entity text.
+    :paramtype text: str
+    :keyword offset: Required. The starting index of this entity in the query.
+    :paramtype offset: int
+    :keyword length: Required. The length of the text.
+    :paramtype length: int
+    :keyword confidence_score: Required. The entity confidence score.
+    :paramtype confidence_score: float
     """
 
     _validation = {
         'category': {'required': True},
-        'confidence_score': {'required': True, 'maximum': 1, 'minimum': 0},
+        'text': {'required': True},
+        'offset': {'required': True},
+        'length': {'required': True},
+        'confidence_score': {'required': True},
     }
 
     _attribute_map = {
         'category': {'key': 'category', 'type': 'str'},
+        'text': {'key': 'text', 'type': 'str'},
+        'offset': {'key': 'offset', 'type': 'int'},
+        'length': {'key': 'length', 'type': 'int'},
         'confidence_score': {'key': 'confidenceScore', 'type': 'float'},
     }
 
     def __init__(
         self,
         *,
         category: str,
+        text: str,
+        offset: int,
+        length: int,
         confidence_score: float,
         **kwargs
     ):
-        super(DeepstackClassification, self).__init__(**kwargs)
+        super(DeepstackEntity, self).__init__(**kwargs)
         self.category = category
+        self.text = text
+        self.offset = offset
+        self.length = length
         self.confidence_score = confidence_score
 
 
-class DeepstackEntity(msrest.serialization.Model):
-    """The entity extraction result of a LUIS Deepstack project.
+class DeepstackIntent(msrest.serialization.Model):
+    """The intent classification result of a LUIS Deepstack project.
 
     All required parameters must be populated in order to send to Azure.
 
-    :param category: Required. The entity category.
-    :type category: str
-    :param text: Required. The predicted entity text.
-    :type text: str
-    :param offset: Required. The starting index of this entity in the query.
-    :type offset: int
-    :param length: Required. The length of the text.
-    :type length: int
-    :param confidence_score: Required. The entity confidence score.
-    :type confidence_score: float
+    :keyword category: Required. A predicted class.
+    :paramtype category: str
+    :keyword confidence_score: Required. The confidence score of the class from 0.0 to 1.0.
+    :paramtype confidence_score: float
     """
 
     _validation = {
         'category': {'required': True},
-        'text': {'required': True},
-        'offset': {'required': True},
-        'length': {'required': True},
-        'confidence_score': {'required': True},
+        'confidence_score': {'required': True, 'maximum': 1, 'minimum': 0},
```

Comment:
@mohamedshabanofficial does the removal of length, text etc make sense? i.e., did the swagger get updated to remove this? sorry can't quite remember where the swagger you're using is from

Reply:
i got the latest swagger from [here](https://github.com/Azure/azure-rest-api-specs/blob/feature/cognitiveservices/language/specification/cognitiveservices/data-plane/Language/preview/2021-07-15-preview/analyzeconversations.json)

Reply:
- repo: https://github.com/Azure/azure-rest-api-specs 
  - branch: feature/cognitiveservices/language
  - path: /specs/cognitiveservices/language

Reply:
ok these changes make sense!